src/constants/index.ts

```diff
@@ -596,10 +598,15 @@ export const SUGGESTED_BASES: ChainTokenList = {
   ],
   [ChainId.BTTC]: [...WETH_ONLY[ChainId.BTTC], DAI[ChainId.BTTC], USDC[ChainId.BTTC], USDT[ChainId.BTTC]],
   [ChainId.VELAS]: [...WETH_ONLY[ChainId.VELAS], USDC[ChainId.VELAS], USDT[ChainId.VELAS]],
-  [ChainId.OASIS]: [...WETH_ONLY[ChainId.OASIS], USDC[ChainId.OASIS], USDT[ChainId.OASIS]],
+  [ChainId.OASIS]: [
+    ...WETH_ONLY[ChainId.OASIS],
+    USDC[ChainId.OASIS],
+    new Token(ChainId.OASIS, '0x6Cb9750a92643382e020eA9a170AbB83Df05F30B', 6, 'USDT', 'Tether USD'),
```

Comment:
update this at `USDT` const

Reply:
USDT[OASIS] already have default value
<img width="436" alt="image" src="https://user-images.githubusercontent.com/33005392/192983924-c594a44b-f803-43aa-9446-70a24b89f1c2.png">

Reply:
Which one is the most transfer/holder? We should choose one and make it on common base.

Reply:
we have a list defined from product.

Reply:
update it into USDT const

Reply:
I see 2 places use it, so I don't want to edit it. 
<img width="405" alt="image" src="https://user-images.githubusercontent.com/33005392/192987288-22b90873-3e49-4c4c-aa09-2724e3b732e8.png">

Reply:
Just do it, it make no sense if we use different USDT in different places in website. It's like cheating user in some way.